File: src/IsbInterop.CSharp/Tasks/Wrappers/ControlJobInfo.cs
namespace IsbInterop.Tasks.Wrappers
{
  /// <summary>
  /// Обертка над IControlJobInfo.
  /// </summary>
  internal class ControlJobInfo : CustomJobInfo, IControlJobInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIControlJobInfo">COM-объект IControlJobInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public ControlJobInfo(object rcwIControlJobInfo, IScope scope)
      : base(rcwIControlJobInfo, scope) { }
  }
}

File: src/IsbInterop/Tasks/Wrappers/ControlJobInfo.vb
Namespace Tasks.Wrappers

  ''' <summary>
  ''' Обертка над IControlJobInfo.
  ''' </summary>
  Friend Class ControlJobInfo
    Inherits CustomJobInfo
    Implements IControlJobInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwIControlJobInfo">COM-объект IControlJobInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwIControlJobInfo As Object, scope As IScope)
      MyBase.New(rcwIControlJobInfo, scope)
    End Sub
  End Class
End NameSpace